packages/forklift-console-plugin/src/modules/Plans/modals/PlanCutoverMigrationModal.tsx

```diff
@@ -139,7 +137,7 @@ export const PlanCutoverMigrationModal: React.FC<PlanCutoverMigrationModalProps>
       isLoading={isLoading}
       isDisabled={!isDateValid || !isTimeValid}
     >
-      {t('Set cutover')}
+      {cutoverSet ? t('Edit cutover') : t('Schedule cutover')}
```

Comment:
this and row 152 reapets itself, and I also see it PlanActionDropdownItems.tsx file, consider extracting it to a const or a util